src/ds/d500/d500-auto-calibration-handler.h

```diff
@@ -0,0 +1,66 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "d500-private.h"
+#include <src/hw-monitor.h>
+
+
+namespace librealsense
+{
+    enum class d500_calibration_state
+    {
+        RS2_D500_CALIBRATION_STATE_IDLE = 0,
```

Comment:
Can we shorten literal names for `enum class` - use will be like `d500_calibration_state::IDLE`

Reply:
tbd

Reply:
done